src/assets/crownheights.json

```diff
@@ -4,7 +4,7 @@
       "type": "Feature",
       "properties": {
         "fill": "#1a4357",
-        "id": "ne"
+        "id": "NE"
```

Comment:
capitalize this so it will be consistent with Airtable values, which are case sensitive.